include/wasm.hh

```diff
@@ -180,24 +180,37 @@ public:
 
 // Ownership
 
-template<class T> using own = std::unique_ptr<T>;
+template <typename T>
+class Destroyer {
```

Comment:
```suggestion
class destroyer {
```
(Keep upper-case names for domain types.)

Reply:
Done.